### PR TITLE
Add release note for 8459

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change adds a release note for the issue described in #8459.

### CODE REVIEW

**Comment by MaxWang-MS** [comment]
I am wondering if this should go into the breaking change section or known issues section. I don't have a strong opinion but just want to bring that up.

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ See [ARFoundation](CrossPlatform/UsingARFoundation.md#install-required-packages)
 
 Some asmdef files are changed and are now only supporting Unity 2018.4.13f1 or later. Compilation erros will show up when upating to MRTK 2.5 in earlier versions of Unity. This can be fixed by going to `Assets\MRTK\Providers\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.asmdef` in the project window and removing the missing reference in the inspector. Repeat those steps with `Assets\MRTK\Providers\Oculus\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef` and `Assets\MRTK\Providers\WindowsMixedReality\XRSDK\Microsoft.MixedReality.Toolkit.Providers.XRSDK.WMR.asmdef`. Note you must revert the changes by replacing those three asmdef files with original (i.e. unmodified) ones when upgrading to Unity 2019.
 
+**CameraCache may create a new camera on shutdown**
+
+In some specific situations, it is possible for the CameraCache to re-create the MainCamera on shutdown. Please see [this issue](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8459) for more information.
```
Comment (wiwei) [inline Documentation/ReleaseNotes.md:123]: Seems like the issue is mostly specific to leap right? Might be worth calling out the context in which it happens

Comment (wiwei) [inline Documentation/ReleaseNotes.md:123]: Seems like the issue is mostly specific to leap right? Might be worth calling out the context in which it happens.

Also as @MaxWang-MS  calls out, it's probably better in the known issues.

This isn't actually a breaking change - it's not a regression or related AFAIK to the 2.5 payload.

Comment (davidkline-ms) [reply Documentation/ReleaseNotes.md:123]: i can do that

**Comment by davidkline-ms** [comment]
>known issues section. 

Good catch @MaxWang-MS! I'll move it to the correct section

File: Documentation/ReleaseNotes.md (hunk 1 above)
Comment (davidkline-ms) [reply Documentation/ReleaseNotes.md:123]: thanks folks! I will make the change now

Comment (davidkline-ms) [reply Documentation/ReleaseNotes.md]: both done!

File: Documentation/ReleaseNotes.md [hunk 2]
```diff
@@ -183,6 +183,10 @@ Motion Controller simulation is now offered in editor play mode along side the e
 
 ### Known issues
 
+**CameraCache may create a new camera on shutdown**
+
+In some situations (ex: when using the LeapMotion provider in the Unity Editor), it is possible for the CameraCache to re-create the MainCamera on shutdown. Please see [this issue](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8459) for more information.
```
Comment (MaxWang-MS) [inline Documentation/ReleaseNotes.md:188]:
```suggestion
In some situations (e.g. when using the LeapMotion provider in the Unity Editor), it is possible for the CameraCache to re-create the MainCamera on shutdown. Please see [this issue](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8459) for more information.
```

**Review by MaxWang-MS — APPROVED** [review]
Looks good to me!

**Pull request merged**